src/Handler/StreamHandler.php

```diff
@@ -284,7 +284,7 @@ function () use ($context, $params) {
 
         return $this->createResource(
             function () use ($request, &$http_response_header, $context) {
-                $resource = fopen($request->getUri(), 'r', null, $context);
+                $resource = fopen($request->getUri(), 'r', '', $context);
```

Comment:
The signature for fopen is: `resource fopen ( string $filename , string $mode [, bool $use_include_path = false [, resource $context ]] )`.

Using `''` for `bool $use_include_path = false` doesn't make sense. `false` would be a better change.